src/seedu/addressbook/AddressBook.java

```diff
@@ -32,6 +32,11 @@
 public class AddressBook {
 
     /**
+     * The total number of arguments allowed.
+     */

```

Comment:
Are there benefits to putting all the global constants (those that have a scope for the entire application) in a separate class? Think about it. What type of class should it be? What should your access be in that case?